.travis.yml

```diff
@@ -17,6 +17,8 @@ before-script:
 script:
     - python manage.py test
     - coverage run --source="authors" manage.py test&&coverage report -m
+env:
+    - SECRET_KEY = '7pgozr2jn7zs_o%i8id6=rddie!*0f0qy3$oy$ttt3'
```

Comment:
@mariamiah Please remove this. I have added it to the `Travis` environment.

Reply:
okay, was  testing if i would correct the travis error